README.md

```diff
@@ -838,6 +838,22 @@ oneview_user 'User1' do
 end
 ```
 
+### oneview_scope
+
+Scope resource for HPE OneView.
```

Comment:
Make a note that this only exists for the API300 module. Put that same note in the example file too. Wouldn't hurt to add some more comments in the example either.